src/app/inventory/item-move-service.ts

```diff
@@ -883,7 +883,7 @@ function ItemService(): ItemServiceType {
         storeService.getStores();
       const storeId = store.id;
       options.triedFallback = true;
-      const reloadedStore = reloadedStores.find((s) => s.id === storeId);
+      const reloadedStore = (reloadedStores as DimStore[]).find((s) => s.id === storeId);
```

Comment:
This assertion is unnecessary since it does not change the type of the expression  @typescript-eslint/no-unnecessary-type-assertion